### ISSUE TITLE
Improve student's ability to share cert

### ISSUE THREAD

**Issue by ryancwalsh** [NEAR-Edu/near-certification-tools#66]
As @rashaabdulrazzak suggested, the email that a student receives about their cert should link to the cert page (rather than the account page) so that the cert is immediately shareable from there.

1. We can remove this sentence from the email: `You can see all of your NEAR University certificates at https://certificates.near.university/account/rashaabdulrazzak.near.`
2. The minting API endpoint returns the tokenId. The URL of the cert page just relies on the tokenId. We should change the hyperlink of the image in the email to point to the cert page (such as https://certificates.near.university/certificate/7300d49067054a7397aff186b4af6a7d)
3. We should update that cert page to fetch the details of the cert from on-chain (yes, unfortunately it will require a second fetch in addition to the fetch of the on-chain details for generating the image) so that we know the account of the recipient, and we can add a link so that the visitor can easily click into that "account" page to view all certs of that student.

**Comment by ryancwalsh** [comment]
@petarvujovic98 Actually, is it easy for you to do the first two steps today? We can leave the third step for after launch.

**Comment by petarvujovic98** [comment]
I can do the first two steps today easily

**Comment by ryancwalsh** [comment]
Yeah wait. I need to delete Esin's and Rasha's mainnet certs, and we need to merge branches into `main` and redeploy etc and then re-run the earlier batches.

But first, I want you @petarvujovic98 and/or @encody to look at https://github.com/NEAR-Edu/near-certification-tools/pull/65 and https://github.com/NEAR-Edu/near-certification-tools/pull/67 and review them enough to feel confident that we should proceed.

**Comment by ryancwalsh** [comment]
Ok, I've deleted all certs from mainnet and redeployed. See https://github.com/NEAR-Edu/near-certification-tools/issues/46

For *this* issue, what remains (not a high priority, and can definitely wait til after launch) is:

1. We should update that cert page to fetch the details of the cert from on-chain (yes, unfortunately it will require a second fetch in addition to the fetch of the on-chain details for generating the image) so that we know the account of the recipient, and we can add a link so that the visitor can easily click into that "account" page to view all certs of that student.
2. From Esin: "On account/example.near where all the certificates are listed, I think there could be a title for each one of them so that the user can navigate and find the certificate they are looking for easier but that's just me :)"
